### ISSUE TITLE
Can't locate Moo.pm in @INC

### ISSUE THREAD

**Issue by eserte** [sanko/Finance-Alpaca#1]
It seems that Moo is not listed as a prerequisite. On some of my smoker systems I see failures like this:
```
#   Failed test 'use Finance::Alpaca;'
#   at t/00_compile.t line 5.
#     Tried to use 'Finance::Alpaca'.
#     Error:  Can't locate Moo.pm in @INC (you may need to install the Moo module) (@INC contains: ../lib ./lib ... /usr/local/lib/perl5/5.30/mach /usr/local/lib/perl5/5.30 .) at lib/Finance/Alpaca.pm line 3.
# BEGIN failed--compilation aborted at lib/Finance/Alpaca.pm line 3.
# Compilation failed in require at t/00_compile.t line 5.
# BEGIN failed--compilation aborted at t/00_compile.t line 5.
# Looks like you failed 1 test of 1.
t/00_compile.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

**Comment by eserte** [comment]
Also possibly undeclared: Mojo::Promise. As this is part of (newer?) Mojolicious, you want also declare something like `Mojolicious => 9` or so.

**Comment by sanko** [comment]
Already fixed in c6f4b4655424aca0b4e8c5906d50c4097367d2ed yesterday morning. Just wanted to proofread some of the docs again before releasing it on PAUSE.
